pkg/controller/operators/catalog/operator.go

```diff
@@ -2230,15 +2231,15 @@ func validateExistingCRs(dynamicClient dynamic.Interface, gr schema.GroupResourc
 			return fmt.Errorf("error creating validator for schema version %s: %s", version, err)
 		}
 		gvr := schema.GroupVersionResource{Group: gr.Group, Version: version, Resource: gr.Resource}
-		crList, err := dynamicClient.Resource(gvr).List(context.TODO(), metav1.ListOptions{})
-		if err != nil {
-			return fmt.Errorf("error listing resources in GroupVersionResource %#v: %s", gvr, err)
-		}
-
-		// validate each CR against this version schema
-		for _, cr := range crList.Items {
-			err = validation.ValidateCustomResource(field.NewPath(""), cr.UnstructuredContent(), validator).ToAggregate()
+		pager := pager.New(pager.SimplePageFunc(func(opts metav1.ListOptions) (runtime.Object, error) {
+			return dynamicClient.Resource(gvr).List(context.TODO(), metav1.ListOptions{})
```

Comment:
```suggestion
			return dynamicClient.Resource(gvr).List(context.TODO(), opts)
```

Reply:
The `ListOptions` argument `opts` isn't being used...

Reply:
Resolved in [ca452be](https://github.com/operator-framework/operator-lifecycle-manager/pull/3387/commits/ca452bec6f199a9c686c9581c83d6ef5008b14b1)